autodeploy.sh

```diff
@@ -34,16 +34,22 @@ git clone $REPO --single-branch --branch $TARGET_BRANCH out
 
 pushd out
 # Remove everything then recreate the content, to avoid keeping stale files
-git rm -rf dist index.html API_Doc
+git rm -rf dist index.html API_Doc examples/layers examples/GUI
 popd
 # Copy build results
 cp -R dist out/
 # generate the API documentation
 npm run doc -- -d out/API_Doc
 # Copy demo
 cp index.html out/
+# Copy layers
+cp -R examples/layers out/
```

Comment:
Won't that create an `out/layers/` rather than `out/examples/layers/`? (as expected by the `git rm` at lines 37 and 52)